client/app/startup/SearchPageApp.js

```diff
@@ -79,17 +79,16 @@ export default (props) => {
     'sign_up',
   ], { locale });
 
-  const bootstrappedData = TransitImmutableConverter.fromJSON(props.searchPage.data);
+  const bootstrappedData = TransitImmutableConverter.fromJSON(_.get(props, 'searchPage.data', null));
 
-  const rawListings = bootstrappedData
-    .get(':data');
+  const rawListings = _.get(bootstrappedData, ':data', []);
```

Comment:
These are already converted to Immutable.js format and it would be better to handle gets etc. through immutable's own functions (those immutable objects are quite far away from normal JS objects and therefore lodash doesn't fully know what it's doing with those maps/lists etc.)

There is a default for Immutable.get too
https://facebook.github.io/immutable-js/docs/#/Map/get